# transform from vicon-checkerboard-body to sensor-checkerboard-body
T_VCB_CB:
- [0.0, 1.0, 0.0, 0.04971]
- [1.0, 0.0, 0.0, 0.04971]
- [0.0, 0.0, -1.0, 0.0]
- [0.0, 0.0, 0.0, 1.0]

#transform from pointcloud-body to sensor-body
T_PB_SB:
- [0.999971, -0.00454671, -0.00605791, -0.0588283]
- [0.00468742, 0.999715, 0.0234204, -0.000282071]
- [0.00594969, -0.0234481, 0.999707, -0.000145381]
- [0.0, 0.0, 0.0, 1.0]

#transform from sensor-vicon-body to pointcloud-body
T_VSB_PB:
- [0.999083,   0.0255534,  -0.0343537,    0.110879]
- [0.0354644,   -0.943448,    0.329617,   0.0301386]
- [-0.0239881,   -0.330533,   -0.943489, -0.00535208]
- [         0,           0,           0,           1]
